kotlin/authentication-profile/src/main/java/com/esri/arcgisruntime/samples/authenticationprofile/MainActivity.kt

```diff
@@ -43,8 +43,10 @@ class MainActivity : AppCompatActivity() {
         val portal = Portal("http://www.arcgis.com", true)
 
```

Comment:
Fix to logic makes sense to me. Just a heads up that I get 'Portal failed to load' toast when I try running the authentication profile app

Reply:
This is working OK for me with latest daily build. Can you repro the same issue with the Java version, same build?

Reply:
Ok, its working for me now on the latest build. 'Portal failed to load' was on build 1890.

The formatting is different for me between kotlin and java.